starter-code/server.js

```diff
@@ -56,14 +74,23 @@ app.post('/articles', function(request, response) {
 
   function queryThree(author_id) {
     client.query(
-      ``, // TODO: Write a SQL query to insert the new article using the author_id from our previous query
-      [], // TODO: Add the data from our new article, including the author_id, as data for the SQL query.
+      `INSERT INTO articles (title, author, "authorUrl", category, "publishedOn", body)
+       WHERE title=$1, author=$2, "authorUrl"=$3, category=$4, "publishedOn"=$5, body=$6
+       author_id=$7;`, // TODO: MAYBE DONE Write a SQL query to insert the new article using the author_id from our previous query
+      [ author_id.body.title,
```

Comment:
also, author_id isn't an object. What is the object that has all the request data on it?